tasks/config.yml

```diff
@@ -12,6 +12,19 @@
     _bbb_upgrade.changed
   notify: restart bigbluebutton
 
+- name: Update /etc/bigbluebutton akka configs
+  become: true
+  template:
+    src: "{{ item }}"
+    dest: "/etc/bigbluebutton/{{ item }}"
+    owner: bigbluebutton
+    group: root
+    mode: '0600'
+  loop:
+    - bbb-apps-akka.conf
+    - bbb-fsesl-akka.conf
+  notify: restart bigbluebutton
+
```

Comment:
Needed since the single / multi kurento setup feature also enables setting a `esl_password` => `{{ bbb_freeswitch_socket_password }}` for the freeswitch.

Noticed I forgot to add the `freeswitch.yml` from the tasks. next commit will add it.